manpage edits, mispellings and a feature.

gapan opened the pull request:
Hi,

this pull request does three things:
1. It adds the missing --no-gtk-update-icon-cache switch in the manpage
2. It adds the install/slack-desc and install/doinst.sh files in the package database, if they are present in the package. That brings it in line with the original pkgtools and opens the possibility that someone can search for packages that have a doinst script for example (grep doinst.sh /var/log/packages/*)
3. It fixes several typos and mispellings.

Please review and pull if you like.

Commit:
Edited "cange" typo to "change"

Commit:
Edited "Direcory" typo to "Directory"

Commit:
Update the man page for the --no-gtk-update-icon-cache option.

Commit:
Make whitespace in pkgdb the same as with Slackware's installpkg.

Commit:
Report sizes the same way slackware's pkgtools do.
Sizes smaller than 1 MB are reported in KBs, with no decimals.
Sizes smaller than 10 MB are reported in MBs, with one decimal.
Sizes larger than 10 MB are reported in MBs, with no decimals.

Commit:
Edited "direcotry" typo to "directory"

Commit:
Show the slack-desc and doinst.sh files in the package database, just
like the slackware pkgtools do.

Commit:
Report package sizes correctly with --list, however they are listed in
the package database, in KB, or MB.

Commit:
Package sizes in MB may have a decimal point, so a float needs to be
used instead of an int when parsing the package database.

Commit:
Revert all changes about package sizes commited by me

Commit:
Apply patch by megous for reporting package information the same way as with the slackware tools

gapan committed:
Upstream merge

gapan committed:
Put back missing backslash in comment border

gapan committed:
Fix typo and add missing switch in SYNOPSIS

The --no-ldconfig switch was mispelled as --no-ldconfigs.
Also add the missing --no-gtk-update-icon-cache switch.

gapan committed:
Add support for brotli packages

Support handling .tbr packages, which are compressed with the new Google
compression algorithm, brotli:
https://github.com/google/brotli

gapan committed:
Merge remote-tracking branch 'upstream/master'

gapan committed:
Bump version to 1.0s2

Commit:
Apply commit 64bf55a of megous/spkg

Fix howto page link do dloads.php

gapan committed:
Merge pull request #1 from DidierSpaier/master

Apply commit 64bf55a of megous/spkg

gapan committed:
Merge branch 'master' of github.com:gapan/spkg

gapan committed:
Remove block for package upgrades

Some packages, such as aaa_elflibs and aaa_base were blacklisted for
upgrades. That goes against what pkgtools do. All blacklisting code has
been removed, it should be possible to upgrade any package now.

gapan committed:
Bump version to 1.0s3

ghost commented:
Hi George. It looks like you've defacto taken over the maintainership of the project. If you'd like I can send you the source code for the website and point the spkg.megous.com subdomain to your server, or perhaps give you ftp access, or do redirect to some website you control, if you'd like to take over completely.

gapan commented:
Hi Ondřej. Sure, I don't mind. Whatever you feel more comfortable with. The website can also be put on github pages I guess. I would like for you to stay with this project since you know the code a lot better than I do, but I understand if you're not interested in it anymore.

gapan committed:
Extract code to separate functions

Extract the ldconfig and gtk cache update code into their own functions.
They may be used in a single place, but this makes the code so much
clearer.

gapan committed:
Add vim modelines

gapan committed:
Add .gitignore

Ignore files generated by autoconf as well as vim backup files.

gapan committed:
Fix breakage when upgrading glibc-solibs

Do not remove symlinks to critical libc libraries when upgrading. The symlinks
are taken care of when the doinst.sh script is executed anyway, so there is no
harm.

gapan committed:
Update with my details

gapan committed:
Bump version to 1.1

gapan committed:
Add NEWS for 1.1 release

gapan committed:
Fix misplaced parenthesis
